backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/DefaultIOSInput.java

```diff
@@ -643,7 +645,8 @@ public boolean isPeripheralAvailable (Peripheral peripheral) {
 		if (peripheral == Peripheral.Compass) return compassSupported;
 		if (peripheral == Peripheral.OnscreenKeyboard) return true;
 		if (peripheral == Peripheral.Pressure) return pressureSupported;
-		if (peripheral == Peripheral.HardwareKeyboard) return hadHardwareKeyEvent;
+		if (peripheral == Peripheral.HardwareKeyboard)
+			return (Foundation.getMajorSystemVersion() >= 14) ? GCKeyboard.getCoalescedKeyboard() != null : hadHardwareKeyEvent;
```

Comment:
Why put the parens around it?

Reply:
I just got used to write it this way because I am using Kotlin on other projects. I can remove it here.